jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBPreparedStatement.java

```diff
@@ -238,54 +249,54 @@ public void setBoolean(int parameterIndex, boolean x) {
 
   @Override
   public void setByte(int parameterIndex, byte x) throws SQLException {
-    throw new SQLException(METHOD_NOT_SUPPORTED_STRING);
+    throw new SQLException(Constant.PARAMETER_SUPPORTED);
   }
 
   @Override
   public void setBytes(int parameterIndex, byte[] x) throws SQLException {
-    throw new SQLException(METHOD_NOT_SUPPORTED_STRING);
+    throw new SQLException(Constant.PARAMETER_SUPPORTED);
   }
```

Comment:
In IoTDB, we have a `Binary` class that contains a byte array. Can we use it to implement this method?